rpc-control.go

```diff
@@ -26,13 +27,19 @@ import (
 const (
 	controlRPCPath = reservedBucket + "/control"
 	healPath       = controlRPCPath + "/heal"
+	shutdownPath   = controlRPCPath + "/shutdown"
 )
 
 // Register control RPC handlers.
 func registerControlRPCRouter(mux *router.Router, objAPI ObjectLayer) {
+	fmt.Printf("registering rpc\n")
 	healRPCServer := rpc.NewServer()
 	healRPCServer.RegisterName("Heal", &healHandler{objAPI})
 	mux.Path(healPath).Handler(healRPCServer)
+
+	shutdownRPCServer := rpc.NewServer()
+	shutdownRPCServer.RegisterName("Shutdown", &shutdownHandler{})
```

Comment:
Let's not create new handlers for each. Move this to 'controllerAPIHandlers' and make this behave similar to web-router.go